test/extensions/RunInScope.test.js

```diff
@@ -51,11 +51,13 @@ describe("global Run function", () => {
         }
     });
 
-    it("returns invalid for runtime errors", () => {
+    it("throws an exception for runtime errors", () => {
         fs.readFileSync.mockImplementationOnce(() => `sub main(): _ = {}: _.crash(): end sub`);
-        expect(RunInScope.call(interpreter, new BrsString("pkg:/errors/exec.brs"))).toBe(
-            BrsInvalid.Instance
-        );
+        try {
+            RunInScope.call(interpreter, new BrsString("pkg:/errors/exec.brs"));
+        } catch (err) {
+            expect(err.message).toBe("'crash' is not a function and cannot be called.");
+        }
```

Comment:
Interestingly, this is actually not the behavior we want here!  Both the global `Run` and the custom `_brs_.runInScope` need to return `invalid`, or tests via `@hulu/roca` will abort immediately upon the first syntax error, causing malformed [tap](https://testanything.org/) output and broken formatting into the Mocha "spec" format.

This test breaking in 95ba0ac2 caught that change! 🎉

Reply:
OK